src/components/player/index.js

```diff
@@ -143,7 +144,7 @@ export default class Player {
 
       let canvases = helper.getCanvases();
       if (canvases.length > 1) {
-        initCanvasNavigation(canvases);
+        console.error('ERROR: too many canvases');
```

Comment:
Would this cause manifests with multiple canvas elements not to work anymore? As there is no init for those but rather a console error?